load.php

```diff
@@ -31,3 +33,15 @@
 	];
 	Altis\register_module( 'search', __DIR__, 'Search', $options, __NAMESPACE__ . '\\bootstrap' );
 } );
+
+// Ensure ElasticPress commands don't use debug mode as it can result
+// in out of memory errors on big sites as queries are logged during bulk updates.
+add_action( 'altis.loaded_autoloader', function () {
+	if ( ! defined( 'WP_CLI' ) || ! WP_CLI || ! isset( $_SERVER['argv'] ) ) {
+		return;
+	}
+	// phpcs:ignore HM.Security.ValidatedSanitizedInput -- Comparing $_SERVER value to a string directly.
+	if ( ( $_SERVER['argv'][1] ?? '' ) === 'elasticpress' ) {
```

Comment:
`wp --url=foo elasticpress ...` won't work with this afaik? I think there's a WP CLI helper to get this.